ci_framework/roles/libvirt_manager/molecule/deploy_layout/converge.yml

```diff
@@ -48,6 +48,8 @@
           nets:
             - public
             - osp_trunk
+          extra_disks_num: 1
```

Comment:
care to add some assertion? like "volume exists in pool", "volume exists on disk", "volume is attached to vm" and so on?

Reply:
Sure. They are added in the latest commit.